oci/oci-explanation/ubuntu-oci-container.rst

```diff
@@ -0,0 +1,22 @@
+.. _ubuntu-oci-container-images:
+
+Ubuntu OCI container images
+===========================
+
+The Open Container Initiative (OCI) establishes standards for constructing container 
+images that can be reliably installed across a variety of compliant host environments.
+
+Ubuntu’s `LTS Docker Image Portfolio <https://ubuntu.com/security/docker-images>`_ 
+provides OCI-compliant images that receive stable security updates and predictable 
+software updates, thus ensuring consistency in both maintenance schedule and operational 
+interfaces for the underlying software your software builds on.
+
+Moreover, `Ubuntu-based containers <https://ubuntu.com/containers>`_ (like
+chiselled container images) all leverage the Ubuntu container image as their
+starting point, and thus also profit from its support and security commitments.
+
+The Ubuntu container image is built from a minimal rootfs tarball. This tarball
+is similar to `Ubuntu Base <https://wiki.ubuntu.com/Base>`_ 
```

Comment:
These old wiki pages are going away "soon" :tm: and from what I've heard Ubuntu Base is mainly used internally. We will probably just want to remove the reference to Ubuntu Base altogether, otherwise we're expecting users to be familiar with it and chances are they won't be.

Reply:
Fair enough. Removing